spec/views/group_members/index.html.erb_spec.rb

```diff
@@ -2,25 +2,21 @@
 
 RSpec.describe "group_members/index", type: :view do
   let(:organisation) { build(:organisation, slug: "Department for testing group members") }
-  let(:user1) { build(:user, organisation:) }
-  let(:user2) { build(:user, organisation:) }
   let(:group) { create(:group, name: "Group 1", organisation:) }
   let(:add_editor) { false }
 
+  let(:rows) { [OpenStruct.new({ name: "user1", email: "user1@gov.uk", role: :editor, actions: [] })] }
```

Comment:
Would it be possible to also test whether a `:delete` action has triggered the button appearing?

Reply:
I've changed the tests to check for this.